### PR TITLE
Remove deprecated terminal option "-e"

### PR DESCRIPTION
**Pull request by boppy**
Replacing deprecated `-e` option from `gnome-terminal`.

Also using the user's default shell (`$SHELL`; if set) after script execution with fallbach to "bash" if `SHELL` is not set.

See #52

### CODE REVIEW

**Comment by p-e-w** [comment]
Thank you for the PR. Since when is this syntax for `gnome-terminal` available? I aim to maintain compatibility with GNOME 3.14.

File: argos@pew.worldwidemann.com/menuitem.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ var ArgosMenuItem = new Lang.Class({
           } else {
             // Run bash immediately after executing the command to keep the terminal window open
```
Comment (p-e-w) [inline argos@pew.worldwidemann.com/menuitem.js]:
```suggestion
            // Run shell immediately after executing the command to keep the terminal window open
```

**Comment by boppy** [comment]
The question is, if this is a version dependent thing:

> There is no '--' option as such; '-- ' is the GNU standard way to denote the *end* of the options.
>
> *source: https://bugzilla.gnome.org/show_bug.cgi?id=774595* (C. Persch; committer/maintainer at gnome-terminal)

[This is the commit](https://gitlab.gnome.org/GNOME/gnome-terminal/commit/ad4edbd11874007bd73de29bfb2cedb8054644d1) the depr-messages have been added, but I cannot find any changes related to this.

Nevertheless I took the binary from [this 3.14.2 RPM Package](https://rpmfind.net/linux/rpm2html/search.php?query=gnome-terminal(x86-64)) and it's working on my local.

Because my local arch was not able to extract I used a ubuntu-docker container for these first steps:
```
mkdir /tmp/gnome-trmnl-test
cd /tmp/gnome-trmnl-test
wget "https://rpmfind.net/linux/mageia/distrib/5/x86_64/media/core/release/gnome-terminal-3.14.2-1.mga5.x86_64.rpm"
rpm2cpio gnome-terminal-3.14.2-1.mga5.x86_64.rpm | cpio -idmv
```
You'll find the binary at `/tmp/gnome-trmnl-test/usr/bin/gnome-terminal`. I copied the binary to my local machine (`docker cp argos-ubu:/tmp/gnome-trmnl-test/usr/bin/gnome-terminal ~/argos-dev/gtrmnl`) and ran

```
./gtrmnl --version
# returns "GNOME Terminal 3.14.2"
./gtrmnl -- bash -c "echo 'working'; exec $SHELL"
```
*I changed the name of the binary to avoid any PATH collisions*

**Comment by p-e-w** [comment]
Merged! Thank you for taking the time to test this properly.

I hope you don't mind that I squashed the 4 commits into one.

**Pull request merged**